src/odemis/acq/align/test/autofocus_test.py

```diff
@@ -56,20 +56,9 @@ class TestAutofocus(unittest.TestCase):
     """
     Test autofocus functions
     """
-    backend_was_running = False
-
     @classmethod
     def setUpClass(cls):
-
-        try:
-            testing.start_backend(SECOM_CONFIG)
-        except LookupError:
-            logging.info("A running backend is already found, skipping tests")
-            cls.backend_was_running = True
-            return
-        except IOError as exp:
-            logging.error(str(exp))
-            raise
+        testing.start_backend(SECOM_CONFIG)
```

Comment:
This refactor is unrelated and not described in the PR description. Is this error handling superseded by something internal of start backend?

Reply:
It's the last line of the PR description. It's also explicitly in a separate commit.

Reply:
Ah my bad! All good :heavy_check_mark: